content/kubeone/main/guides/addons/_index.en.md

```diff
@@ -192,6 +192,19 @@ addons:
       key2: value2
 ```
 
+### Well-known Parameters
+
+Some embedded addons can be configured via "well-known" parameters. They are documented below.
```

Comment:
Can we please create a follow up issue in this or kubeone repo to document all well-know parameters? There are in fact many of them and I think it's a great idea to have them documented in a central place like this.

Reply:
Created #1518.

Reply:
Thank you!